### DISCUSSION TITLE
Can I dynamically add data directories in minio

### DISCUSSION THREAD

**Discussion by jzdayz** [minio/minio#11392] [Q&A] [answered]
## Expected Behavior

My data directory disk is full, and I hope to add a directory to facilitate expansion

## Current Behavior

I don't know how to achieve it

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used (`minio --version`): minio version RELEASE.2020-04-02T21-34-49Z
* Server setup and configuration: 
```txt
export MINIO_ACCESS_KEY=xxx
export MINIO_SECRET_KEY=xxx

/opt/minio/minio server --config-dir /etc/minio \
http://xxx/minio/data1 http://xxx/minio/data2  \
http://xxx/minio/data1 http://xxx/minio/data2 \
http://xxx/minio/data1 http://xxx/minio/data2 \
```
* Operating System and version (`uname -a`):
Linux zsbx2 3.10.0-1062.el7.x86_64 #1 SMP Wed Aug 7 18:08:02 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

**Comment by jzdayz** [comment]
Can i just add configuration, like this 
```
export MINIO_ACCESS_KEY=xxx
export MINIO_SECRET_KEY=xxx

/opt/minio/minio server --config-dir /etc/minio \
http://xxx1/minio/data1 http://xxx1/minio/data2  \
http://xxx2/minio/data1 http://xxx2/minio/data2 \
http://xxx3/minio/data1 http://xxx3/minio/data2 \
http://xxx4/minio/data1 http://xxx4/minio/data2 \
```

**Comment by klauspost** [comment]
Please ask questions on community Slack: https://slack.min.io/

(Adding another zone will only work when servers are defined as ellipses: `minio server /data{1...12}`)

**Comment by jzdayz** [comment]
> Please ask questions on community Slack: https://slack.min.io/
> 
> (Adding another zone will only work when servers are defined as ellipses: `minio server /data{1...12}`)

It's not working , i update the run command .
```
export MINIO_ACCESS_KEY=xxx
export MINIO_SECRET_KEY=xxx

/opt/minio/minio server --config-dir /etc/minio \
http://xxx1/minio/data1 http://xxx1/minio/data2  http://xxx1/data/data3  \
http://xxx2/minio/data1 http://xxx2/minio/data2 http://xxx2/data/data3 \
http://xxx3/minio/data1 http://xxx3/minio/data2 http://xxx3/data/data3\
```

**Comment by jzdayz** [comment]
I added a disk to my server (three servers in total),I just want to store data in a new disk

**Comment by harshavardhana** [comment, marked as answer]
You cannot add zones in this manner @jzdayz zones have to be added in ellipses syntax. https://github.com/minio/minio/tree/master/docs/distributed#expanding-existing-distributed-setup

Individual command line args is a legacy style to start MinIO follow the latest documentation.
